packages/bpk-tokens/tokens/base.common.js

```diff
@@ -204,6 +204,9 @@ module.exports = {
   colorYellow800: "rgb(194, 142, 0)",
   colorYellow900: "rgb(156, 114, 0)",
   primaryGradient: "linear-gradient(135deg, #00b2d6 0%, #02DDD8 100%)",
+  checkboxCheckedTickColor: "rgb(0, 178, 214)",
+  checkboxDisabledTickBackgroundColor: "rgb(230, 228, 235)",
```

Comment:
Where does this come from?

Reply:
I should remove those I reverted the approach